CLAUDE.md

```
@@ -190,7 +180,42 @@ app/features/{feature}/
+
+- **Arrow function (const)**: Use when component body contains ONLY a return statement with no hooks, variables, or logic
+
+   ```typescript
+   // ✅ CORRECT - Only a return statement
+   export const Badge = ({ children, color }: Props): JSX.Element => (
+     <span className={cn(badgeVariants({ color }))}>{children}</span>
+   )
+   ```
+
+- **Function declaration**: Use when component has ANY of the following:
+   - React hooks (useState, useEffect, useTranslation, etc.)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)